src/components/Background/index.view.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./Background.scss";
+import { ReactComponent as Wire } from "images/wire.svg";
+import { ReactComponent as Rainbow } from "images/rainbowpath.svg";
+interface BackgroundProps {
+  children?: React.ReactNode;
+}
+
+const Background: React.FC<BackgroundProps> = ({
+  children,
+}: BackgroundProps) => {
+  return (
+    <div>
+      <div className="Background">
+        <div className="Background__children">
+          {Array.isArray(children)
+            ? children.filter((child, index) => index < 4)
+            : children}
```

Comment:
A more efficient approach might be to require that `children` is an array, and then conditionally render if it exists instead of using a ternary operator.